go/logic/applier.go

```diff
@@ -89,10 +89,12 @@ func (this *Applier) InitDBConnections() (err error) {
 	if err := this.validateAndReadTimeZone(); err != nil {
 		return err
 	}
-	if impliedKey, err := mysql.GetInstanceKey(this.db); err != nil {
+	if impliedKey, err := mysql.GetInstanceKey(this.db, this.migrationContext.AliyunRDS); err != nil {
 		return err
 	} else {
-		this.connectionConfig.ImpliedKey = impliedKey
+		if this.migrationContext.AliyunRDS != true {
```

Comment:
Please change to `if !isAliyunRDS {`

Reply:
Or also see my suggestion to replace the entire block.

Reply:
Thanks for your suggestion ! replace the entire block makes the code cleaner.